ambry-commons/src/main/java/com.github.ambry.commons/Container.java

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright 2017 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+package com.github.ambry.commons;
+
+import org.json.JSONObject;
+
+
+/**
+ * A representation of a container.
```

Comment:
what do you think is the appropriate place to document what a container itself is? this could be one of the places.